backend/app/assets/stylesheets/spree/backend/shared/_typography.scss

```diff
@@ -17,6 +17,11 @@ hr {
   border-left: none;
 }
 
+strong,
+b {
+  font-weight: 600;
```

Comment:
There should be a Bootstrap variable for this. Or we should add one for this weight.